### ISSUE TITLE
Midi learn metronome parameters

### ISSUE THREAD

**Issue by jpbm18** [zynthian/zynthian-issue-tracking#912]
Currently none of the parameters on the metronome page utilize midi learn.  

I’d like to change tempo BPM with my midi controller, as well as turn metronome on and off with midi controller.

**Comment by riban-bw** [comment]
I agree this should work. This may require an engine to be built or maybe existing engine reused by the Tempo view.
